instrumentation/apache-httpclient/apache-httpclient-5.0/build.gradle

```diff
@@ -0,0 +1,23 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group.set("org.apache.httpcomponents.client5")
+    module.set("httpclient5")
+    versions.set("[5.0,)")
+  }
+}
+
+dependencies {
+  compileOnly project(":bootstrap")
+  compileOnly("org.apache.httpcomponents.client5:httpclient5:5.0")
+
+  compileOnly("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure")
+  compileOnly("io.opentelemetry.instrumentation:opentelemetry-instrumentation-api-semconv")
+  compileOnly("io.opentelemetry.javaagent:opentelemetry-javaagent-tooling")
+
+  // Needs to be "implementation" for ByteBuddy to find a class we instrument
+  implementation("io.opentelemetry.javaagent.instrumentation:opentelemetry-javaagent-apache-httpclient-5.0:${versions.opentelemetryJavaagentAlpha}")
```

Comment:
Does this mean we would include in our classloader _another_ copy of `opentelemetry-javaagent-apache-httpclient-5.0`? That'd be probably very bad.